packages/destination-actions/src/destinations/fullstory/identifyUserV2/index.ts

```diff
@@ -0,0 +1,71 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import { createUserRequestParams } from '../request-params'
+import { normalizePropertyNames } from '../vars'
+
+const action: ActionDefinition<Settings> = {
+  title: 'Identify User V2',
+  description: 'Sets user identity variables, creates user if it does not exist',
+  platform: 'cloud',
+  defaultSubscription: 'type = "identify"',
+  fields: {
+    uid: {
+      type: 'string',
+      required: true,
+      description: "The user's id",
+      label: 'User ID',
+      default: {
+        '@path': '$.userId'
+      }
+    },
+    display_name: {
```

Comment:
Nit: Our other fields are camel-cased e.g. `displayName`. You could rename this field to `displayName` and still pass it to the API request body as `display_name`.